shark_turbine/kernel/wave/utils.py

```diff
@@ -312,7 +312,7 @@ def safe_subs(input: Any, subs: List[Tuple[IndexExpr, IndexExpr]]) -> Any:
     Substitute input using provided `subs` list if input is sympy object.
     Otherwise return input unchanged.
     """
-    if isinstance(input, sympy.Basic):
+    if isinstance(input, sympy.Basic) or isinstance(input, IndexSequence):
```

Comment:
nit: you can do `isinstance(input, (sympy.Basic, IndexSequence))`

Reply:
sure will change.

Reply:
thats pretty cool, didnt know you could do that :)